### PR TITLE
Add error handling slides

### PR DESCRIPTION
**Pull request by nachosoloaga**
This PR includes the slides for error handling with try..catch blocks.

### CODE REVIEW

**Review by JulianPasquale — APPROVED** [review]
Can we add one or two slides about how to create a custom error? https://javascript.info/custom-errors

**Comment by nachosoloaga** [comment]
> Can we add one or two slides about how to create a custom error? https://javascript.info/custom-errors

We totally can, getting to work on that!

File: src/slides/class1.jsx [hunk 1]
```diff
@@ -1258,5 +1258,173 @@ export default () => (
...
+             }
+            `}
+        </CodePane>
+      </div>
+    </Slide>
+
+    <Slide backgroundColor="#000000">
+      <div>
+        <Heading margin="0px" fontSize="50px">
+          Creando errores custom
```
Comment (JulianPasquale) [inline src/slides/class1.jsx:1378]:
```suggestion
          Creando nuestro propio error
```
We avoid avoid spanglish here 😆

File: src/slides/class1.jsx [hunk 2]
```diff
@@ -1258,5 +1258,173 @@ export default () => (
...
+        </Heading>
+        <Text fontSize={30}>
+          Cuando estamos desarrollando es probable que querramos tener nuestras propias clases de errores
+          para reflejar situaciones específicas que pueden fallar durante la ejecución de nuestros programas.
+          Para esto, JavaScript nos permite definir clases que extienden de la clase base <b>Error</b>.
+        </Text>
+        <CodePane language="js">
+            {`
```
Comment (JulianPasquale) [inline src/slides/class1.jsx:1386]: Looks like you have 4 spaces here... You don´t need to fix this

File: src/slides/class1.jsx [hunk 3]
```diff
@@ -1258,5 +1258,173 @@ export default () => (
...
+    </Slide>
+
+    <Slide backgroundColor="#000000">
+      <div>
+        <Heading margin="0px" fontSize="50px">
+          Creando errores custom (cont.)
+        </Heading>
+        <Text fontSize={30}>
+          En el ejemplo anterior, definimos un nuevo tipo de error para reflejar fallos en validaciones. Si bien al
+          extender de la clase <b>Error</b> nuestra clase hereda las 3 propiedades básicas <b>message, name and stack</b>,
```
Comment (JulianPasquale) [inline src/slides/class1.jsx:1409]:
```suggestion
          extender de la clase <b>Error</b> nuestra clase hereda las 3 propiedades básicas <b>message, name y stack</b>,
```

**Review by JulianPasquale — APPROVED** [review]

**Pull request merged**